tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -2625,5 +2723,177 @@ Status StorageManagerCanonical::group_metadata_vacuum(
   return consolidator->vacuum(group_name);
 }
 
+void StorageManager::encryption_key_from_configs(EncryptionKey &enc_key, const Config* config) {
+  if (!config) {
+    config = &config_;
+  }
+  std::string enc_key_str, enc_type_str;
+  bool found_key = false;
```

Comment:
We could just have one bool found = false; value and reuse.

Reply:
ok